scalafmt-cli/src/main/scala/org/scalafmt/cli/CliOptions.scala

```diff
@@ -66,11 +77,19 @@ object CliOptions {
   }
 }
 
+object NoopOutputStream extends OutputStream { self =>
+  override def write(b: Int): Unit = ()
```

Comment:
Would you override two other `write` methods with `()` just in case?
https://docs.oracle.com/javase/8/docs/api/java/io/OutputStream.html

(LIke this https://github.com/apache/commons-io/blob/d3137782060046872e1ca15f244cbe6d4142b7cf/src/main/java/org/apache/commons/io/output/NullOutputStream.java#L36-L67)